lapis/src/main/kotlin/org/genspectrum/lapis/model/mutationsOverTime/MutationsOverTimeModel.kt

```diff
@@ -55,6 +55,8 @@ data class MutationsOverTimeCell(
             "the date range. Confirmed deletions (i.e., \"-\") are included.",
     )
     var coverage: Int,
+    @param:Schema(description = "Total number of sequences in the date range.")
+    var totalCount: Int,
```

Comment:
I think that we should avoid the redundancy of having the count which is the same for each mutation. What about putting it into `dateRanges`?